pkg/virt-launcher/notify-client/client.go

```diff
@@ -597,10 +594,7 @@ func (n *Notifier) SendK8sEvent(vmi *v1.VirtualMachineInstance, severity string,
 			return false, nil
 		}
 
-		ctx, cancel := context.WithTimeout(context.Background(), n.sendTimeout)
-		defer cancel()
```

Comment:
I think we still need it because it uses a different timeout:
```
ctx, cancel := context.WithTimeout(ctx, n.sendTimeout)
```
Same for the above one

Reply:
.